alibi/explainers/ale.py

```diff
@@ -171,13 +184,13 @@ def explain(self, X: np.ndarray, features: List[int] = None, min_bin_points: int
             feature_names=feature_names
         )
 
-    def build_explanation(self,
-                          ale_values: List[np.ndarray],
-                          ale0: List[np.ndarray],
-                          constant_value: float,
-                          feature_values: List[np.ndarray],
-                          feature_deciles: List[np.ndarray],
-                          feature_names: np.ndarray) -> Explanation:
+    def __build_explanation(self,
```

Comment:
Is there a reason for the double-under instead of single-under?

Reply:
No ... I meant single-under. I will update it.